recipes/panic/meta.yaml

```diff
@@ -0,0 +1,104 @@
+{% set name = "panic" %}
+{% set version = "9.4.2" %}
+{% set python_min = "3.9" %}
+
+package:
+  name: panic-suite
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/panic-{{ version }}.tar.gz
+  sha256: f2772142ac60f69a8927a98317213804abb5becff565a852438288f97353827b
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

Comment:
You don't want to create a package for panic-suite so the script shouldn't be here.

If you check the logs, panic and panic-gui are empty.

I think that the way multiple outputs work in conda-build is a bit confusing.
I'd personally recommend to use the new v1 format (with rattler-build). See https://github.com/conda-forge/taurus-feedstock/blob/main/recipe/recipe.yaml

Anyway, if you want to keep the meta.yaml, you need to move this script to the panic output.
You don't need any requirement and test at the global level.

Reply:
Hi @benjee, I added those requirements and test due to conda-forge linter comments